templates/adding_a_new_model/cookiecutter-template-{{cookiecutter.modelname}}/test_modeling_tf_{{cookiecutter.lowercase_modelname}}.py

```diff
@@ -889,69 +889,6 @@ def test_model_common_attributes(self):
                 name = model.get_bias()
                 assert name is None
 
-    def test_resize_token_embeddings(self):
```

Comment:
This test already exists in the common tests 👉 [here](https://github.com/huggingface/transformers/blob/c28d04e9e252a1a099944e325685f14d242ecdcd/tests/test_modeling_tf_common.py#L1147).

This duplicated test was removed in many places in this PR.